cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java

```diff
@@ -144,10 +145,10 @@ private int countConnectedClients() {
   }
 
   private <T> Gauge<T> registerGauge(String name, Gauge<T> gauge) {
-    return Metrics.register(factory.createMetricName(name), gauge);
```

Comment:
What was this existing `CassandraMetricsRegistry.Metrics` doing? Might this impact any of the metrics in the persistence modules?

Reply:
It was wrapping a static `MetricRegistry` instance, and also publishing each metric as a JMX MBean.

I replaced the static registry by the one injected from the health-checker module. I also disable JMX because I don't think we want to do that by default, but that could be easily re-added if necessary.